docs/state-diff/README.md

```diff
@@ -0,0 +1,185 @@
+ 🤖⚡ External Computation with Kontrol ⚡🤖
+=================================
+**Injecting Foundry Execution into Proofs**
+
+This folder is an example for including external computation with Foundry into Kontrol proofs.
+To achieve this, there are several files involved and the process has several steps. If you encounter any problems while following what is described here, please reach out in [Discord](https://discord.gg/CurfmXNtbN), we'll be happy to assist!
```

Comment:
```suggestion
To achieve this, there are several files involved and the process has several steps. If you encounter any problems while following these instructions, please reach out in [Discord](https://discord.gg/CurfmXNtbN), we'll be happy to assist!
```

Reply:
Also, maybe I'd rephrase the first sentence to say something like "This projects contains the files and description of steps needed for this process". :)

Reply:
Liking this as well :)